### PR TITLE
Regenerated files for v3.1

### PR DESCRIPTION
**Pull request by abhi1693**

### CODE REVIEW

**Comment by abhi1693** [comment]
It's been over 3 years that this plugin has been updated with the Netbox API spec. It would be extremely helpful if I can receive some feedback over here to bring this to life.

@humblearner @nmaludy @armab @emedvedev @jjm

File: pack.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ keywords:
   - networking
   - ipam
   - dcim
-version: 2.0.0
+version: 2.1.0
```
Comment (arm4b) [inline pack.yaml]: Considering such a massive diff, new actions, and the breaking changes, it would make more sense to bump the pack version to `3.0.0`

**Review by arm4b — CHANGES_REQUESTED** [review]
Thanks for the PR!

I left one comment to address.
Please also include a changelog in the https://github.com/StackStorm-Exchange/stackstorm-netbox/blob/master/CHANGES.md

**Comment by abhi1693** [comment]
> Thanks for the PR!
> 
> I left one comment to address. Please also include a changelog in the https://github.com/StackStorm-Exchange/stackstorm-netbox/blob/master/CHANGES.md

I have updated the version and the changelog.

File: CHANGES.md [hunk 2]
```diff
@@ -11,9 +11,12 @@
 - Added the `netbox_webhooks` sensor which listens for inbound webhooks from NetBox and fires triggers into StackStorm.
 
 ## 1.0.0
+
+- Drop Python 2.7 support
```
Comment (arm4b) [inline CHANGES.md]: Looking at the history, dropping python2 should go under the v2.0.0 changelog version.
The pack version was released correctly, it's just the CHANGES.md record is misleading.

See: 
https://github.com/StackStorm-Exchange/stackstorm-netbox/commit/4b8444574fe7c324db28a78c8c18eec5fed2e74d#diff-5472355e552efc17032084527cf9f00b21fda4841dc4996f2eca9fe7a5e3ada9

**Review by arm4b — CHANGES_REQUESTED** [review]

**Review by arm4b — APPROVED** [review]
Thanks!

**Pull request merged**